controllers/restore_controller.go

```diff
@@ -106,16 +106,21 @@ func (r *RestoreReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 		return ctrl.Result{}, nil
 	}
 
-	otherRestoreName, err := r.isOtherRestoresRunning(ctx, restore)
+	// don't create restores if there is any other active resource in this namespace
+	activeResourceMsg, err := r.isOtherResourcesRunning(ctx, restore)
 	if err != nil {
 		return ctrl.Result{}, err
 	}
-	if otherRestoreName != "" {
-		msg := "Restore instance " + otherRestoreName + " is currently running, ignoring this request."
-		updateRestoreStatus(restoreLogger, v1beta1.RestorePhaseFinishedWithErrors, msg, restore)
+	if activeResourceMsg != "" {
+		updateRestoreStatus(
+			restoreLogger,
+			v1beta1.RestorePhaseFinishedWithErrors,
+			activeResourceMsg,
+			restore,
+		)
```

Comment:
@birsanv Restore status is set to `RestorePhaseFinishedWithErrors` if there is any active backupschedule/restore in this namespace without reconcile Requeue